README.md

```diff
@@ -205,11 +236,20 @@ creation_rules:
 
 If you prefer having a separate GnuPG key, see [Use with GnuPG instead of ssh keys](#use-with-gnupg-instead-of-ssh-keys).
 
+### 3b. Get a age Public key for your machine
+
+The `sops-ssh-to-age` tool is used to convert any ssh public key to the age format.
+This way you can convert any key:
+```console
+$ nix run -f default.nix sops-ssh-to-age -c sh -c 'ssh-keyscan my-server.com | sops-ssh-to-age'
```

Comment:
Should I quickly package `sops-ssh-to-age` in nixpkgs. I did so for the other tool as well.

Reply:
Yeah, why not

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/136098